pages/blog/whats-new-in-graphql-codegen-v2.mdx

```diff
@@ -0,0 +1,135 @@
+
+## Removal of deprecations
+
+We also used the need for a major version to remove a few deprecations from the codebase.
+
+#### `typescript-resolvers`
+
+In `typescript-resolvers`, we had a generated signature for `IResolvers` and `IDirectiveResolvers` (which were deprecated 2 years ago), `v2` of this plugin is removing these proxy types.
+
+Also, `noSchemaStitching` flag is now set to `true` by default, so the generated resolvers signature is simpler and matches the needs of most projects.
+
+> If you are using Schema Stitching, you can set `noSchemaStitching: false` to keep the old behavior.
+
+#### `typescript-compatibility`
+
+The `typescript-compatibility` plugin was created to make it easier to migrate from v0 to v1 of codegen, and not being actively developed for a few years now.
+
+With this release, we no longer support or develop `typescript-compatibility` plugin, so if you are using it, please consider to migrate your usage and types to the new form.
+
+## New TypeScript Plugin for Operations!
+
+A month ago, [`Maël Nison`](https://github.com/arcanis) reached out to us with [a new concept](https://github.com/arcanis/graphql-typescript-integration) for matching your actual GraphQL operation string and the generated TypeScript types.
+
+With a few adjustments, [`n1ru4l`](https://github.com/n1ru4l) managed to [turn that amazing idea into a plugin and a preset](https://github.com/dotansimha/graphql-code-generator/pull/6267) that extends the behavior of `typescript-operation` and `TypedDocumentNode`, and allow you to generate TypeScript types that magically matches your GraphQL query strings (this also required a few fixes in `graphql-tools`, so now the Loaders in graphql-tools are better!).
+
+#### So what does it means?
+
+Today, GraphQL Codegen scans your codebase and looks for all the operations that are being used by your components (or, from `.graphql` files), then it generates types and wrapped `TypedDocumentNode` for you.
+
+With this new plugin+preset, you can generate typings for your inline `gql` function usages, without having to manually specify import statements for the documents. All you need to do is import your `gql` function and run codegen in watch mode.
+
+```ts
+import { gql } from '@app/gql';
+
+// TweetFragment is a fully typed document node
+const TweetFragment = gql(/* GraphQL */ `
+  fragment TweetFragment on Tweet {
+    id
+    body
+  }
+`);
+
+const TweetsQueryWithFragment = gql(/* GraphQL */ `
+  query TweetsWithFragmentQuery {
+    Tweets {
+      id
+      ...TweetFragment
+    }
+  }
+`);
+```
+
+You can find [the complete documentation, examples and API reference in codegen website](https://www.graphql-code-generator.com/docs/presets/gql-tag-operations).
+
+> Huge thanks to [Maël Nison](https://github.com/arcanis), who conceptualized the foundation for this preset [over here](https://github.com/arcanis/graphql-typescript-integration).
```

Comment:
I would add a section about "What's next?"
To talk about all the new ideas, including the new resolvers PR and type masking and other things